bin/positionalFunctions.js

```diff
@@ -1,17 +1,18 @@
-function getDiPositional(yargs) {
+function defaultPositional(yargs) {
     yargs.positional('playerName', {
         type: 'string',
         default: 'player1',
         describe: 'Player name'
     });
 }
 
+function getDiPositional(yargs) {
+    defaultPositional(yargs);
+}
+
 function addPlayerPositional(yargs) {
-    yargs.positional('playerName', {
-        type: 'string',
-        default: 'player2',
```

Comment:
This default value and the one below is 'player2' instead of 'player1' 
Does changing this have any significance?

Reply:
This won't change anything, default is a moot value now unfortunately

Reply:
@lherlein if moot do you recommend it to be removed?

Reply:
I believe that a default tag is necessary (?) and even if not it's a decent placeholder for now - as it is a supported feature in cliv2. I say keep as is